app/modules/sntp.c

```diff
@@ -48,6 +50,8 @@
 #include "rtc/rtctime.h"
 #endif
 
+struct netif * eagle_lwip_getif(uint8 index);
```

Comment:
I'm not sure this is correct, but I haven't found another way to define it here.